app/helpers/currency_helper.rb

```diff
@@ -1,10 +1,11 @@
 module CurrencyHelper
   def format_currency(amount, currency, tooltip: true)
+    currency_name = I18n.t("common.currency.#{currency.try(:locale_name)}")
```

Comment:
I think we should use`currency.code` for this instead of this extra `locale_name` method -- the currency codes should already uniquely identify the currencies and not have any spaces/other troublesome characters.